vipps-login-api-quick-start.md

```diff
@@ -7,62 +7,59 @@ END_METADATA -->
 
-### Step 1: Get the Postman collection
+### Step 1: Get the Vipps Postman collection and environment
 
-Import the collection by following the steps below:
+Save the following files to your computer:
 
-1. Click `Import` in the upper-left corner.
-2. Import the [vipps-login-api-postman-collection.json](tools/vipps-login-api.postman_collection.json) file.
+* [Vipps Login API Postman collection](tools/vipps-login-api.postman_collection.json)
+* [Vipps Login API Postman environment](tools/vipps-login-api.postman_environment.json)
 
-### Step 2: Import the Postman environment
+### Step 2: Import the Vipps Postman files
 
-1. Click `Import` in the upper-left corner.
-2. Import the [vipps-login-api.postman_environment.json](tools/vipps-login-api.postman_environment.json) file.
+1. In Postman, click *Import* in the upper-left corner.
+1. In the dialog that opens, with *File* selected, click *Upload Files*.
+1. Select the two files you have just downloaded and click *Import*.
 
 ### Step 3: Set up Postman environment
 
 1. Click the down arrow, next to the "eye" icon in the top-right corner, and select the environment you have imported.
 1. Click the "eye" icon and, in the dropdown window, click `Edit` in the top-right corner.
 1. Fill in the `Current Value` for the following fields to get started.
-   - `well-known_uri`
-   - `client_id`
-   - `client_secret`
-   - `redirect_uri`
+   * `client_id` - Merchant key is required for getting the access token.
+   * `client_secret` - Merchant key is required for getting the access token.
+   * `well-known_uri` - URL to well-known endpoint for used environment. In the test environment, you can use <https://apitest.vipps.no/access-management-1.0/access/.well-known/openid-configuration>. See [API Guide: well known](vipps-login-api#openid-connect-discovery-endpoint) for more details.
+   * `redirect_uri` - The URL where the user is sent after finishing a login. The URL must be exactly the same as the one specified on <portal.vipps.no> in the "Setup login" section for your sale unit. For testing, you could, for example, use <http://localhost> in both places.
 
 ## Make API calls
 
 1. Send request `Get OIDC well-known`.
-1. In Postman environment "vipps-login-api" copy value of key `start_login_uri`, and use this url in any browser.
-1. Finish Vipps login.
-1. Copy query param `code` from url the browser was redirected to after finishing Vipps login. Paste this code into key `code` in Postman environment `vipps-login-api`.
-1. Send request `Get token` to get access token and id token.
-1. Send request `Get user info` to get user info of logged in user.
+1. In your active Postman environment, "vipps-login-api", copy the value of key `start_login_uri`, and use this URL in any browser.
+1. Finish Vipps login. This request includes a scope parameter that requests access to user information, so you may see an extra screen during login.
```

Comment:
so you may see an extra screen during login.
 
 Rephrase to
 
 This means that if you have not yet consented to sharing your user information a distinct screen asking for your consent will appear the first time.

Reply:
That's great! Thanks!